src/main/java/com/flightstats/hub/dao/ChannelService.java

```diff
@@ -289,6 +286,12 @@ public void deleteBefore(String channel, ContentKey limitKey) {
         return Optional.ofNullable(channelConfig);
     }
 
+    public boolean isNotLiveChannel(String channelName, boolean checkCache) {
```

Comment:
I'd probably have left this as `isLiveChannel` for readability but I bet IntelliJ thought you should invert it because it's only used in that state. 6 of one, half dozen of the other. 🤷‍♂️